src/AppBundle/Entity/ProcurationProxy.php

```diff
@@ -612,7 +613,7 @@ public function getVoteOffice(): ?string
         return $this->voteOffice;
     }
 
-    public function setVoteOffice(?string $voteOffice)
+    public function setVoteOffice(string $voteOffice): void
```

Comment:
Let me remind you I introduced a form type extension for TextType and its children (Email, Url, Textarea, ...) that defines `empty_data` as string by default ;).